wallet/wallet.go

```diff
@@ -1520,6 +1521,7 @@ b:
 	if err != nil {
 		return nil, err
 	} else {
+		msg=strings.ReplaceAll(msg,"\"","")
```

Comment:
Why we have to replace the msg?

Reply:
The return after sending the transaction is like this "abfc5f36a4e17081fc5167f390e9f0575293892c7cc8e3ba098f55f1a3735b1a" 
Quotes need to be removed

Reply:
Please refactor code after PR15, it's kind of unnecessary data handling because of poorly-designed data exchanging